### PR TITLE
OSDOCS-8805: Adds etcd version query to MicroShift

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.15+

Issue:
[OSDOCS-8805](https://issues.redhat.com/browse/OSDOCS-8805)

Link to docs preview:
[Support book](https://69917--ocpdocs-pr.netlify.app/microshift/latest/microshift_support/microshift-etcd#microshift-version-etcd_microshift-etcd)
[Troubleshooting book](https://69917--ocpdocs-pr.netlify.app/microshift/latest/microshift_troubleshooting/microshift-version#microshift-version-etcd_microshift-version)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/microshift-etcd-version.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
++
+[source,terminal]
+----
+$ microshift-etcd version
+----
++
+.Example output
+[source,terminal,subs="attributes+"]
+----
+{microshift-short}-etcd Version: 4.15.0_0.nightly_arm64_2023_11_20_235936_20231129145639_153b6f993
```
Comment (ggiguash) [inline modules/microshift-etcd-version.adoc:24]: We should not use macros in executable names, please change to hardcoded `microshift-etcd`

File: modules/microshift-etcd-version.adoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
++
+[source,terminal]
+----
+$ microshift-etcd version
+----
++
+.Example output
+[source,terminal,subs="attributes+"]
+----
+microshift-etcd Version: 4.15.0_0.nightly_arm64_2023_11_20_235936_20231129145639_153b6f993
```
Comment (ggiguash) [inline modules/microshift-etcd-version.adoc:24]: Please, change this version to `4.15.1` as well.

File: microshift_support/microshift-etcd.adoc [hunk 3]
```diff
@@ -7,8 +7,10 @@ include::_attributes/attributes-microshift.adoc[]
 toc::[]
 
 [role="_abstract"]
-The etcd service is delivered as part of the {product-title} RPM. The etcd service is run as a separate process and the lifecycle is managed automatically by {product-title}.
+The etcd service is delivered as part of the {product-title} RPM. The etcd service is run as a separate process and the lifecycle is managed automatically by {microshift-short}.
```
Comment (ggiguash) [inline microshift_support/microshift-etcd.adoc:10]:
```suggestion
The etcd service is delivered as part of the {product-title} RPM. The etcd service is run as a separate process and its lifecycle is managed automatically by {microshift-short}.
```

File: modules/microshift-etcd-version.adoc [hunk 4]
```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-version.adoc
+// * microshift_support/microshift-etcd.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-version-etcd_{context}"]
+= Checking the etcd version
+
+To enhance troubleshooting, you can get the version information for the etcd database included with your {microshift-short} version.
```
Comment (ggiguash) [inline modules/microshift-etcd-version.adoc:10]: I'm not sure about this sentence. The bottom line is that we should tell users - here's how you get etcd version.

File: modules/microshift-etcd-version.adoc [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
++
+[source,terminal]
+----
+$ microshift-etcd version
+----
++
+.Example output
+[source,terminal,subs="attributes+"]
+----
+microshift-etcd Version: 4.15.1_0.nightly_arm64_2023_11_20_235936_20231129145639_153b6f993
```
Comment (ggiguash) [inline modules/microshift-etcd-version.adoc:24]:
```suggestion
microshift-etcd Version: 4.15.1
```

**Review by ggiguash — CHANGES_REQUESTED** [review]

File: modules/microshift-etcd-version.adoc [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+----
++
+.Example output
+[source,terminal,subs="attributes+"]
+----
+microshift-etcd Version: 4.15.1
+Base etcd Version: 3.5.10
+----
+
+* To display the full database version information, run the following command using the `-o json` option:
```
Comment (tmalove) [inline modules/microshift-etcd-version.adoc:28]:
```suggestion
* To display the full database version information, run the following command by using the `-o json` option:
```
Suggestion: See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u

**Review by tmalove — APPROVED** [review]

**Pull request merged**